### PR TITLE
Fixes markdown format.

### PR DESCRIPTION
**Pull request by chirag-wagento**
## This PR is a:
- [ ] New topic
- [ ] Content fix or rewrite
- [x] Bug fix or improvement
 

## Summary
 Fixes typo.
When this pull request is merged, it will...

### CODE REVIEW

File: guides/v2.1/comp-mgr/trouble/cman/tshoot_backup.md [hunk 1]
```diff
@@ -13,25 +13,25 @@ functional_areas:
...
+*	[unix.stackexchange.com: Tracking down where disk space has gone on Linux?](http://unix.stackexchange.com/questions/125429/tracking-down-where-disk-space-has-gone-on-linux){:target="&#95;blank"}
 
-<h3 id="trouble-backup-os">Operating system error</h3>
+### Operating system error {#trouble-backup-os}
 Unfortunately, we can't recommend anything specific because of the variety of errors you might encounter. We can suggest, however, you:
 
 *	Contact your system administrator
-*	Search public forums like <a href="http://unix.stackexchange.com" target="_blank">stackexchange</a> or <a href="http://stackoverflow.com" target="_blank">stackoverflow</a>
-*	Open a <a href="https://github.com/magento/magento2/issues" target="_blank">GitHub issue</a> and we'll try to help
+*	Search public forums like [stackexchange](http://unix.stackexchange.com){:target="&#95;blank"} or [stackoverflow](http://stackoverflow.com){:target="&#95;blank"}
```
Comment (shrielenee) [inline guides/v2.1/comp-mgr/trouble/cman/tshoot_backup.md]: Should be "Stack Overflow" and "Stack Exchange"

**Comment by chirag-wagento** [comment]
Hi @shrielenee  I have fixed this text as per your advice. Please check

Thanks

File: guides/v2.1/comp-mgr/upgrader/upgrade-start.md [hunk 2]
```diff
@@ -11,25 +11,24 @@ functional_areas:
   - Upgrade
 ---
 
-<h2 id="compman-prereq">Prerequisites</h2>
-Before continuing, complete all tasks discussed in <a href="{{ page.baseurl }}/comp-mgr/prereq/prereq_compman.html">Prerequisites</a>.
+## Prerequisites {#compman-prereq}
```
Comment (shrielenee) [inline guides/v2.1/comp-mgr/upgrader/upgrade-start.md]: Please complete these changes in the v2.2 version if needed, as its not symlinked.

File: guides/v2.1/comp-mgr/upgrader/upgrade-checklist.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ functional_areas:
   - Upgrade
```
Comment (shrielenee) [inline guides/v2.1/comp-mgr/upgrader/upgrade-checklist.md]: Please complete these changes in v2.2 if needed, as they are not symlinked.

File: guides/v2.1/comp-mgr/upgrader/ce-ee-upgrade-start.md [hunk 4]
```diff
@@ -11,40 +11,38 @@ functional_areas:
   - Upgrade
 ---
```
Comment (shrielenee) [inline guides/v2.1/comp-mgr/upgrader/ce-ee-upgrade-start.md]: Please complete these changes in v2.2 version if needed, as it is not symlinked.

**Comment by shrielenee** [comment]
I forgot to check versioning and symlinking! See my comments about needed changes.

**Comment by chirag-wagento** [comment]
Hello @shrielenee I have fixed this versioning and symlinking as per your advice. Please check

Thanks

**Comment by shrielenee** [comment]
Thanks for quickly making that change @chirag-wagento - merging shortly.

**Comment by shrielenee** [comment]
Before I make this change @chirag-wagento - I want to make sure you didn't add symlinks between pages that didn't already exist, correct? Sometimes, due to changes between versions of Magento, we don't want to symlink all versions of the docs as they require different content. 

Thanks again! Please let me know.

**Comment by chirag-wagento** [comment]
Hello @shrielenee  I was change  for only non symlink file.

**Pull request merged**